src/client/opamPinCommand.ml

```diff
@@ -491,12 +491,9 @@ and source_pin
       version, None
   in
 
-  if not (OpamPackage.has_name st.packages name) &&
-     not (OpamConsole.confirm
-            "Package %s does not exist, create as a %s package?"
-            (OpamPackage.Name.to_string name)
-            (OpamConsole.colorise `bold "NEW"))
-  then raise Aborted;
+  if not (OpamPackage.has_name st.packages name) then
+    OpamConsole.note "Package %s was unknown until now"
```

Comment:
If anyone has a better idea feel free to change it

Reply:
Suggestion
```suggestion
    OpamConsole.note "Package %s does not exist in registered repositories."
```

Reply:
as a user i would find this message confusing. The repositories we refer to here are opam repositories but they might understand source repositories instead since the command does something with a source repository usually.

Reply:
```suggestion
    OpamConsole.note "Package %s does not exist in opam repositories registered in the current switch."
```
?

Reply:
actually i'm not sure. What do we want to say? Maybe something like:
```suggestion
    OpamConsole.note "%s is a new package."
```